src/lib/tzkt/utils.ts

```diff
@@ -0,0 +1,76 @@
+export type ParameterFa12 = {
+  entrypoint: string;
+  value: {
+    to: string;
+    from: string;
+    value: string;
+  };
+};
+interface Fa2Transaction {
+  to_: string;
+  amount: string;
+  token_id: string;
+}
+interface Fa2OpParams {
+  txs: Fa2Transaction[];
+  from_: string;
+}
+export type ParameterFa2 = {
+  entrypoint: string;
+  value: Fa2OpParams[];
+};
+export type ParameterLiquidityBaking = {
+  entrypoint: string;
+  value: {
+    target: string;
+    quantity: string; // can be 'number' or '-number
+  };
+};
+
+export function isTzktOperParam(param: any): param is {
+  entrypoint: string;
+  value: any;
+} {
+  if (param == null) return false;
+  if (typeof param.entrypoint !== 'string') return false;
+  return 'value' in param;
+}
+
+export function isTzktOperParam_Fa12(param: any): param is ParameterFa12 {
+  if (!isTzktOperParam(param)) return false;
+  if (param.value == null) return false;
+  if (typeof param.value.to !== 'string') return false;
+  if (typeof param.value.from !== 'string') return false;
+  if (typeof param.value.value !== 'string') return false;
+
+  return true;
+}
+
+/**
+ * (!) Might only refer to `param.entrypoint === 'transfer'` case
```

Comment:
yes, because there are literally two types of standartified smartcontracts for tokens with proper documentation and support in libraries such as taquito or in indexeres/API's. We can support more operations for non-standart tokens/smartcontracts (such as tezos-domains, liquidity baking, liquidity providers, bridge-tokens, wrap-tokens), but currently that was out of scope of token's wallet and more about scope of acitivity tracker/indexer.